operations/observability/mixins/workspace/rules/components/workspaces/alerts.libsonnet

```
@@ -73,6 +73,23 @@
               gitpod_workspace_regular_not_active_percentage > 0.15 AND sum(gitpod_ws_manager_workspace_activity_total) > 100
             |||,
           },
+          {
+            alert: 'GitpodWorkspacesNotStarting',
+            labels: {
+              severity: 'critical',
+            },
+            'for': '10m',
+            annotations: {
+              runbook_url: 'https://github.com/gitpod-io/runbooks/blob/main/runbooks/GitpodWorkspaceNotStarting.md',
+              summary: 'workspaces are not starting',
+              description: 'inactive regular workspaces exists but workspaces are not being started',
+            },
+            expr: |||
+              gitpod_workspace_regular_not_active_percentage > 0.0
+              AND
+              sum(rate(gitpod_ws_manager_workspace_startup_seconds_bucket{type="REGULAR",le="+Inf"}[10m])) == 0
```

```suggestion
              sum(rate(gitpod_ws_manager_workspace_startup_seconds_count{type="REGULAR"}[10m])) == 0
```

Every prometheus histogram comes with a counter 🙂